lib/generators.gradle

```diff
@@ -0,0 +1,22 @@
+dependencies {
+    implementation 'io.swagger.core.v3:swagger-annotations'
+    runtimeOnly 'org.webjars.npm:swagger-ui-dist:4.9.0'
+    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli'
+
+    // Versioned by Spring:
+    implementation 'javax.validation:validation-api'
+    implementation 'org.webjars:webjars-locator-core'
+
+    annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
+}
+
+def artifactGroup = "${group}.javapfb"
+
+// see https://github.com/n0mer/gradle-git-properties
+gitProperties {
+    keys = []
+    customProperty('javapfb.version.gitTag', { it.describe(tags: true) })
```

Comment:
This is used if you have git.properties loaded somewhere in your app. You may want to write some code that shows that it works by adding a `--version` command to your CLI.

Reply:
I'll take a look at this with the next ticket where I'm going to set up publishing.